src/Components/RoverGalleryImage/RoverGalleryImage.tsx

```diff
@@ -0,0 +1,31 @@
+﻿import React, {useState, useEffect} from "react";
+import {getRoverImages, RoverImages} from "../ApiFetch/RoversApiFetch";
+
+interface RoverGalleryImageProps {
+    dateInput : string | null;
+}
+
+export function RoverGalleryImage(props : RoverGalleryImageProps) {
+    const [roverImage, setRoverImage] = useState<RoverImages | null>(null);
+    
+    useEffect(() => {
+        getRoverImages(props.dateInput)
+            .then(image => setRoverImage(image));
+    }, [props.dateInput]);
+
+    if (!roverImage) {
+        return <div>Loading</div>
+    }
+    
+    return (
+        <div>
+            <img data-testid={"image-from-rover"} src={roverImage?.img_src} alt=""/>
+            <p>Sol: {roverImage?.sol}</p>
+            <hr/>
+            <hr/>
+            <hr/>
+            <hr/>
```

Comment:
We can probably get rid of all of these
(can use CSS if we need some space!)

Reply:
Done!